dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -126,30 +128,32 @@ protected boolean isGithubAppEnabledOnRepositoryWithRenovateApi(String fullRepoN
             HttpURLConnection conn = (HttpURLConnection) url.openConnection();
             conn.setRequestMethod("GET");
             conn.setRequestProperty("Accept", "application/json");
-            conn.setRequestProperty("authorization", "Bearer " + appServerApiToken);
+            conn.setRequestProperty("Authorization", appServerApiToken);
             conn.connect();
+
+            log.info("HTTPS Response Code: " + conn.getResponseCode());
+            log.info("HTTPS Response Message: " + conn.getResponseMessage());
             
-            Integer responseCode = conn.getResponseCode();
-            if (responseCode != 200) {
-                throw new RuntimeException("HttpResponseCode: " + responseCode);
-            } else {
-                String inline = "";
-                Scanner scanner = new Scanner(url.openStream());
-                while (scanner.hasNext()) {
-                    inline += scanner.nextLine();
-                }
-                scanner.close();
+            if (conn.getResponseCode() == HttpURLConnection.HTTP_OK) {
+                BufferedReader in = new BufferedReader(new InputStreamReader(conn.getInputStream()));
+                String inputLine;
+                StringBuilder response = new StringBuilder();
 
-                //Using the JSON simple library parse the string into a json object
-                JSONParser parse = new JSONParser();
-                JSONObject dataObject = (JSONObject) parse.parse(inline);
-                String appInstallationState = (String) dataObject.get("state");
-                String appActivationStatus = (String) dataObject.get("status");
+                while ((inputLine = in.readLine()) != null) {
+                    response.append(inputLine);
+                }
+                in.close();
 
-                // Return app installation and activation status
-                if (appInstallationState == "installed" && appActivationStatus == "activated") {
+                if (response.toString().contains("installed")) {
```

Comment:
We need to check for 'activated' as well.

Reply:
@AvvariSaiBharadwaj The reason I don't add `activated` is because there are some different activation statuses for a repo that is onboarded to Renovate, for example:
```
{"id":"a632a388-4147-52d0-b328-3a564d1a7876","name":"falcon-summercamp-python-933","fullName":"falcon-codelabs/falcon-summercamp-python-933","state":"installed","status":"onboarded","installedAt":"2024-08-22 19:42:46.76875","pullRequestStats":{"merged":0,"open":1,"ignorhanelliotphan@hanelli-ltmgc24 Desktop % 
```

Reply:
@AvvariSaiBharadwaj this has been replaced with simpler HttpClient